### PR TITLE
Resolution of RPM package warnings and errors spotted by rpmlint 

### PR DESCRIPTION
**Pull request by clebergnu**
As a preparation for proposing official Avocado packages to Fedora, let's make sure our packages are free or warnings/errors (as detected by rpmlint).

### CODE REVIEW

**Comment by clebergnu** [comment]
[Propose Avocado packages to Fedora](https://trello.com/c/idiQ3z8M/861-propose-avocado-packages-to-fedora)

File: avocado/utils/software_manager.py [hunk 1]
```diff
@@ -991,3 +989,7 @@ def install_distro_packages(distro_pkg_map, interactive=False):
 
     elif action == 'show-help':
         parser.print_help()
+
+
+if __name__ == '__main__':
+    main()
```
Comment (ldoktor) [inline avocado/utils/software_manager.py]: Well the most appropriate solution would be to implement `__main__.py`, but as it's a library inside the same dir as other libraries, this works fine. And if this passes the checkers, we can use it...

File: avocado/core/version.py [hunk 2]
```diff
@@ -24,6 +22,3 @@
     VERSION = "unknown.unknown"
 
 MAJOR, MINOR = VERSION.split('.')
-
-if __name__ == '__main__':
```
Comment (ldoktor) [inline avocado/core/version.py]: OK, as we don't need it anymore, I'm fine with this change...

File: avocado.spec [hunk 3]
```diff
@@ -4,7 +4,7 @@
 %endif
 %global shortcommit %(c=%{commit}; echo ${c:0:7})
 
-Summary: Avocado Test Framework
+Summary: Framework with tools and libraries for Automated Testing
```
Comment (ldoktor) [inline avocado.spec]: Not bad ;-)

File: examples/testplans/release.json [hunk 4]
```diff
@@ -12,6 +12,9 @@
 	{"name": "Avocado RPM build",
 	 "description": "On your development machine, build the Avocado RPM packages using: `$ make rpm`. Expected result: SRPM and RPM files at `BUILD/RPM/avocado-x.y.z-r.distro.{src,noarch}.rpm`"},
 
+	{"name": "Avocado RPM lint",
+	 "description": "With the built RPMs, check if there are no packaging errors or warnings by running: `$ rpmlint BUILD/RPM/avocado-x.y.z-r.distro.noarch.rpm`.  Expected result: 1 packages and 0 specfiles checked; 0 errors, 0 warnings."},
```
Comment (ldoktor) [inline examples/testplans/release.json]: What is the difference between `spec` file and `rpm` file check? Shouldn't we check both? (I noticed it says `1 packages and 0 specfiles checked; 0 errors, 0 warnings.` and when I try `rpmlint avocado.spec` it also reports no errors.

Also how about adding `make rpmcheck` target?

**Comment by clebergnu** [comment]
@ldoktor the checks on the SPEC file are usually simpler, since no expansions occurred.  I'm all in for a more automated and comprehensive check, such as you suggested with `make rpmcheck`.  This is a small first step though.

**Comment by ldoktor** [comment]
OK, thanks, merged.

**Pull request merged**